### DISCUSSION TITLE
Can we make a field in documents local(or private) which is not synced between nodes?

### DISCUSSION THREAD

**Discussion by bglgwyng** [pouchdb/pouchdb#8536] [Q&A]
In PouchDB, every field in every document is synced between nodes.
However, I want to make a field not to be synced and let each node use this field for their _local_ usage.
> To provide the context, I'm writing a cloud photo gallery using PouchDB and trying to save the local path of each photo for each machine syncing to the cloud.

Of course, this can be achieved via cooperation with another KV store in each node.
But it would be really great if there is a cheap workaround or a concrete solution only with PouchDB.
Do we have any?

**Comment by endyjasmi** [comment]
Best solution I can think of is using `_local` document.
